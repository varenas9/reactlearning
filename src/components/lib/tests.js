import renderer from 'react-test-renderer';

export function render(component) {
  return renderer.create(component);
}