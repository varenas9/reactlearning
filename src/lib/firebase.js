import firebase from 'firebase/app';
import 'firebase/firebase-database';

const config = {
  apiKey: "AIzaSyBbjHJq2E7EaR3mLUssr4j1yl2cOT7SRMQ",
  authDomain: "to-do-list-98c18.firebaseapp.com",
  databaseURL: "https://to-do-list-98c18.firebaseio.com",
  projectId: "to-do-list-98c18"
};
firebase.initializeApp(config);

export const db = firebase.database();

export default firebase;