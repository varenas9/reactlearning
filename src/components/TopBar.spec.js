import React from 'react';
import {render} from './lib/tests';
import TopBar from './TopBar';

describe('TopBar test', ()=>{
  it('should render the Top Bar', () => {
    const component =render(<TopBar/>);
    expect(component.toJSON()).toMatchSnapshot();
  })
})