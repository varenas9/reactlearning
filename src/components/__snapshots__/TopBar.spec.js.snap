// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar test should render the Top Bar 1`] = `
<div
  className="TopBar-root-1"
>
  <header
    className="MuiPaper-root-11 MuiPaper-elevation4-17 MuiAppBar-root-2 MuiAppBar-positionStatic-6 MuiAppBar-colorPrimary-9"
  >
    <div
      className="MuiToolbar-root-38 MuiToolbar-regular-40 MuiToolbar-gutters-39"
    >
      <h6
        className="MuiTypography-root-42 MuiTypography-h6-59 MuiTypography-colorInherit-71"
      >
        To Do List
      </h6>
    </div>
  </header>
</div>
`;
